code/index.html

```diff
@@ -1,12 +1,73 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project Name</title>
-    <!-- dont forget the link to the css file -->
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <!-- Build your site up as you want it! Make sure to use the form element and try to send that form to httpbin by adding action="https://httpbin.org/anything" and method="POST" to your form tag. -->
-  </body>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Nadelöhr_form</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <form action="http://httpbin.org/anything" method="POST">
+        <h2>Bezahlung</h2>
+
+        <fieldset>
+
+            <label>
+                <span>Vorname</span>
+                <input type="text" name="Vorname" placeholder="Jenny" required>
```

Comment:
Nice with the placeholders! When we have many attributes like this, it's nice to split them up for better readability:
```
<input
  required
  type="text"
  name="Vorname"
  placeholder="Jenny">
```